airbyte-featureflag/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-library`
+    kotlin("jvm") version "1.8.0"
+}
+
+dependencies {
+    annotationProcessor(platform(libs.micronaut.bom))
+    annotationProcessor(libs.bundles.micronaut.annotation.processor)
+
+    implementation(platform(libs.micronaut.bom))
+    implementation(libs.micronaut.inject)
+    implementation("com.jayway.jsonpath:json-path:2.7.0")
+    implementation("com.launchdarkly:launchdarkly-java-server-sdk:6.0.1")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.14.0")
```

Comment:
We should add these in to the deps.toml, as there is already a managed jackson version in there.  This will prevent us getting out of sync.

Reply:
👍🏻 I was already planning on doing this